File: app/src/main/java/io/github/zwieback/familyfinance/business/chart/service/formatter/YearValueFormatter.java
package io.github.zwieback.familyfinance.business.chart.service.formatter;

import com.github.mikephil.charting.formatter.ValueFormatter;

public class YearValueFormatter extends ValueFormatter {

    @Override
    public String getFormattedValue(float year) {
        return String.valueOf((int) year);
    }
}

File: app/src/main/java/io/github/zwieback/familyfinance/business/chart/service/formatter/YearValueFormatter.kt
package io.github.zwieback.familyfinance.business.chart.service.formatter

import com.github.mikephil.charting.formatter.ValueFormatter

class YearValueFormatter : ValueFormatter() {

    override fun getFormattedValue(year: Float): String {
        return year.toInt().toString()
    }
}
